### ISSUE TITLE
Problem with qlik license assignment ls

### ISSUE THREAD

**Issue by danielbenoni3C** [qlik-oss/qlik-cli#30]
Hi, i'm new to this community so, if this is the wronge place for this, please, just tell me.

I'm working on a script and when I execute the command `qlik license assignment ls`, it only return 20 lines, and i have around of 133 users. This could be a bug?

If I add the tag --limit with the exact number os registries, it works fine.

**Comment by wennmo** [comment]
Hi @danielbenoni3C, `qlik license assignment ls` will by default only return 20 items, since that is the default value for the Licenses API. https://qlik.dev/apis/rest/licenses#%23%2Fentries%2Fv1%2Flicenses%2Fassignments-get

**Comment by danielbenoni3C** [comment]
Hi @wennmo, thanks for your help. I will look for some workaround.

**Comment by wennmo** [comment]
Hi @danielbenoni3C, as you mentioned you can use the `--limit` flag if you want more results than the default limit of the API. For example `qlik license assignment ls --limit 1000` would return <=1000 assignments, depending on the number of license assignments for your tenant. 

It does however not guarantee that you will get all assignments configured, if the number of assignments is higher than the limit you provide.

If you want to paginate yourself in a script, you can also use the raw command e.g. `qlik raw get v1/licenses/assignments --query "limit=100" --raw`. The response will include the pagination link to the next page for more assignments (if any).

**Comment by danielbenoni3C** [comment]
@wennmo I tested this first but there was an error when handling the result. Now that I understand Powershell a little better, I analyzed the result and identified that the Json result does not end with the character "]", which causes an error using ConvertFrom-Json. I needed to concatenate the character manually, but it worked. Thanks again. 

```
$userLicenseSaaSTemp = qlik license assignment ls --limit 10000

$userLicenseSaaSTemp += "]"

$userLicenseSaaS = $userLicenseSaaSTemp | ConvertFrom-Json
```

**Comment by wennmo** [comment]
@danielbenoni3C Thanks, that sounds like a bug in the qlik-cli and we will look into why the response is not valid JSON. Can you check which version of the qlik-cli you are running i.e running `qlik version`?

We did a bug fix for that particular command in version 2.16.0 that was released last week.

**Comment by danielbenoni3C** [comment]
I'm using the version 2.15.0. I will update and try again for a test.

**Comment by danielbenoni3C** [comment]
It worked on 2.16.0. Thanks @wennmo

**Comment by aihazm** [comment]
fixed in `v2.16.0`
